packages/app/src/components/DBRowSidePanelHeader.tsx

```diff
@@ -63,6 +70,41 @@ export default function DBRowSidePanelHeader({
     [bodyExpanded, mainContent],
   );
 
+  const headerRef = useRef<HTMLDivElement>(null);
+  const [headerHeight, setHeaderHeight] = useState(0);
+  useEffect(() => {
+    if (!headerRef.current) return;
+
+    const updateHeight = () => {
+      if (headerRef.current) {
```

Comment:
not sure you need this safeguard since the null check is on the first line.

Since the ResizeObserver only "works" when an element is in the DOM and is resized, I think it's a safe assumption that it's not null.